code/modules/fabrication/_fabricator.dm

```diff
@@ -31,14 +31,8 @@
 
 	var/list/stored_material
 	var/list/storage_capacity
-	var/list/base_storage_capacity = list(
-		/decl/material/solid/metal/steel =     SHEET_MATERIAL_AMOUNT * 20,
-		/decl/material/solid/metal/aluminium = SHEET_MATERIAL_AMOUNT * 20,
-		/decl/material/solid/metal/copper =    SHEET_MATERIAL_AMOUNT * 20,
-		/decl/material/solid/fiberglass =      SHEET_MATERIAL_AMOUNT * 10,
```

Comment:
Technically a difference here in that all of these now have 20x sheet capacity, but seems insignificant to me.